Tests/FluentDarkModeKitTests/DarkModeKitTests.swift

```diff
@@ -51,7 +55,12 @@ final class DarkModeKitTests: XCTestCase {
     let view = UIView()
     view.backgroundColor = color
     view.tintColor = color
-    XCTAssertFalse(view.backgroundColor === color)
+    if #available(iOS 13.0, *) {
+      XCTAssertTrue(view.backgroundColor === color)
+    }
+    else {
+      XCTAssertFalse(view.backgroundColor === color)
+    }
```

Comment:
I don't think asserting whether the colors are the same reference is necessary. Probably comparing their rgba values after overriding user interface style via `overrideUserInterfaceStyle` property of `UIView` is more appropriate here.

Thoughts? @eyvallah @kylinroc

Reply:
I think here it is testing whether the backgroundColor is DynamicColorProxy or not, since we swizzle setBackgroundColor under iOS 13.0, but on iOS 13+ we use UIColor itself

Reply:
In that case, I think it would be better to use `XCTAssert(view.backgroundColor is DynamicColorProxy)`

Reply:
I think I described it in a wrong way, on iOS 12, color variable is DMDynamicColorProxy (which got passed in the backgroundColor setter), view.backgroundColor (getter) is a UIColor